conf-template/starport.conf

```diff
@@ -1,13 +1,16 @@
 krux.starport {
 
+    # (Optional) make sure all pipelines are scheduled in the configured region
+    region = null
```

Comment:
do we need to update hyperion here? otherwise, setting this to `null` will fallback to `us-east-1` instead of default.

Reply:
This may not be an issue at all but I think that we should do what @talbright suggested above or update the hyperion dependency to incorporate that bug fix. Otherwise a user who uses older version of hyperion and sets `region = null` may encounter that bug again.

Reply:
This would overwrite hyperion regions, so there might be a legitimate use case that user actually want to spin pipelines in different regions?

Reply:
We could discuss this offline or in a separate issue to make it required configuration, for now I'm just going to merge this.